internal/ui/exec_test.go

```
@@ -0,0 +1,30 @@
+	execTerminalProcess(cmd, nil)
+	if cmd.Stdout != os.Stdout {
+		t.Fatal("interactive child stdout was not pinned to the terminal")
+	}
+}
+
+func TestExecTerminalProcessPreservesConfiguredStdout(t *testing.T) {
+	file, err := os.CreateTemp(t.TempDir(), "stdout")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer file.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `file.Close` is not checked

(errcheck)